### PR TITLE
[wasm] Add more PackedSimd methods, logic ops

### PR DESCRIPTION
**Pull request by radekdoulik**

### CODE REVIEW

**Review by vargaz — APPROVED** [review]

**Comment by radekdoulik** [comment]
the CI failures are unrelated

**Comment by akoeplinger** [comment]
@radekdoulik I think the ios-arm64 and tvos-arm64 ones are related.

File: src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Wasm/PackedSimd.cs [hunk 1]
```diff
@@ -854,6 +854,350 @@ public abstract class PackedSimd
         [Intrinsic]
         public static Vector128<nuint>  And(Vector128<nuint>  left, Vector128<nuint>  right) => And(left, right);
 
+        /// <summary>
+        ///   v128.or
+        /// </summary>
+        public static Vector128<sbyte>  Or(Vector128<sbyte>  left, Vector128<sbyte>  right) => Or(left, right);
```
Comment (akoeplinger) [inline src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Wasm/PackedSimd.cs:860]: are these not missing `[Intrinsic]` ?

**Comment by radekdoulik** [comment]
> @radekdoulik I think the ios-arm64 and tvos-arm64 ones might be related, at least we started seeing #85345 that we didn't see in last night's build.

I think I removed accidentally one of the ops on arm64. The above mentioned PR should fix it.

**Pull request merged**